src/style.css

```diff
@@ -1 +1,72 @@
 /* Styles go here */
+html {
+  --color: #00acdc;
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  display: flex;
+  padding: 0 50px;
+  justify-content: space-between;
+  align-items: center;
+  box-shadow: 0 2px 4px rgb(0 0 0 / 5%);
+  border-radius: 4px;
+}
+
+.header,
```

Comment:
You already have styles for the header above. Move margin there and place styles for body near the html